packages/button-theme/__tests__/DatePicker.test.tsx

```diff
@@ -1,9 +1,10 @@
 import { axe, toHaveNoViolations } from 'jest-axe';
 import { render } from '@testing-library/react';
 import React from 'react';
-import DatePicker from '../src/DatePicker';
+import DatePicker, { datepickerStyles } from '../src/DatePicker';
 import '@testing-library/jest-dom/extend-expect';
 import 'regenerator-runtime/runtime';
+import { changeSelectorToObject } from '../../bcgov-theme/utils/test-helpers';
```

Comment:
Might be good to move the `test-helpers` somewhere more general if we're going to be using them in multiple packages, but that can be future tech debt consideration